Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -44,12 +43,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             config.backgroundQueueSecondsDelay = Double(self.storage.bgQDelay ?? "30") ?? 30
             config.backgroundQueueMinNumberOfTasks = Int(self.storage.bgNumOfTasks ?? "10") ?? 10
             config.autoTrackScreenViews = self.storage.isTrackScreenEnabled ?? true
+
+            config.autoTrackPushEvents = true
+
             if let trackUrl = self.storage.trackUrl, !trackUrl.isEmpty {
                 config.trackingApiUrl = trackUrl
             }
         }
 
-        // Add event listeners for in-app. This is not to initialise in-app but event listeners for in-app.
+        // Initialize messaging features after initializing Customer.io SDK
         MessagingInApp.initialize(eventListener: self)
         MessagingPushAPN.initialize { config in
```

Comment:
Looks like the above comment is more suitable at this initialisation method.

`Initialize messaging features after initializing Customer.io SDK`